internal/core/legacy/model/audit.go

```diff
@@ -48,6 +48,35 @@ type AuditSettings struct {
 	Config OrgConfig
 }
 
+type ListAuditParam struct {
+	// +required 查询事件起始时间
```

Comment:
translate to english, or just ignore it if simple enough.

Reply:
has been fixed